patterns/2-structured/early-and-fluid-quality-management-for-predictable-delivery.md

```diff
@@ -0,0 +1,60 @@
+## Title
+
+Early and fluid quality management for predictable delivery
+
+## Patlet
+
+Testing practices are often short-cutted to meet deadlines leading to stability questions regarding the code. Frequent code review and testing is embedded in a smaller pull-request culture.
+
+## Problem
+
+Code stability depends on how confident we are on our tests. In some cases, proper testing is not taking place until later stages (even pre-production deployment), generating delays in the delivery of the releases. Part of the problem is the size of changes. Bigger changes are more difficult to review and test. In other words, our development code is sent to pre-production without passing the tests, creating blocks and loops against pre-production environments while fixing the issues.
+
+The lack of automated testing coverage leaves a lot of room for fears of breaking things (regressions), which in turn tend to prevent contributions and their acceptance. The contributions are also more error-prone and thus need more corrective effort. This delays them and increases the risk of conflicting with other code changes.
+
+## Forces
```

Comment:
```suggestion
## Context
* Proper testing is not taking place until the late stages
* There's fear of regressions for the parts that were once tested. This blocks contribution and innovation because contributors fear breaking working software, and verification is effort-consuming and difficult to schedule for maintainers.
* Testing finds severe and deeply embedded bugs that need significant changes
* Release delays are usual and big

## Forces
```

Reply:
Will already commit this, assuming that you know the Context of this pattern a lot better than me :)